.github/workflows/dev-build-node-cpp-bindings.yml

```diff
@@ -269,7 +213,7 @@ jobs:
       if: ${{ inputs.run_tests }}
       run: |
         cd ts-test;
-        npm install;
+        npm ci;
```

Comment:
Is it possible to add `--ignore-scripts` here?

Reply:
Added --ignore-scripts and ran npm scripts manually without using npm run.